app/views/planning_applications/_all_planning_applications_table.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="govuk-tabs__panel" id="all">
```

Comment:
Adding this as a separate view for now, but I'm about to dry up all the tabs in https://github.com/unboxed/bops/pull/717!